app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/contracts/ActivityContract.kt

```diff
@@ -0,0 +1,24 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation.mvp.contracts
+
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+import io.reactivex.Observable
+
+interface ActivityContract {
```

Comment:
the name of this contract it too generic, please rename to find more easily find it . The same in the DialogFragmentContract please.

Reply:
Done!